sample/src/main/java/com/canhub/cropper/sample/camera_java/app/SCameraFragmentJava.java

```diff
@@ -0,0 +1,317 @@
+package com.canhub.cropper.sample.camera_java.app;
+
+import android.Manifest;
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.content.ActivityNotFoundException;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.Color;
+import android.net.Uri;
+import android.os.Bundle;
+import android.os.Environment;
+import android.provider.MediaStore;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.activity.result.ActivityResultLauncher;
+import androidx.activity.result.contract.ActivityResultContracts;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.core.content.FileProvider;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentActivity;
+import androidx.viewpager.widget.ViewPager;
+
+import com.canhub.cropper.CropImage;
+import com.canhub.cropper.CropImageView;
+import com.canhub.cropper.sample.SCropResultActivity;
+import com.canhub.cropper.sample.camera_java.domain.CameraEnumDomainJava;
+import com.canhub.cropper.sample.camera_java.domain.SCameraContractJava;
+import com.canhub.cropper.sample.camera_java.presenter.SCameraPresenterJava;
+import com.example.croppersample.R;
+import com.example.croppersample.databinding.FragmentCameraBinding;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+import static android.graphics.Color.RED;
+import static android.graphics.Color.WHITE;
+
+public class SCameraFragmentJava extends Fragment implements SCameraContractJava.View {
+
+    public static final int CODE_PHOTO_CAMERA = 811917;
+    static final String DATE_FORMAT = "yyyyMMdd_HHmmss";
+    static final String FILE_NAMING_PREFIX = "JPEG_";
+    static final String FILE_NAMING_SUFFIX = "_";
```

Comment:
I think this is not used, can delete

Reply:
It is used in "$FILE_NAMING_PREFIX$$FILE_NAMING_SUFFIX"

But I confuse why there are two dollar sign in the middle?
I'll use it with java String